ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -447,6 +449,21 @@ function upgrade() {
     echo "Upgrading cluster to ${TARGET} gets started..."
 }
 
+# Log abnormal cluster status
+function dump_status_if_unexpected() {
+    # expecting oc to equal TARGET_MINOR_VERSION, skip if less than .16
+        if [[ -n "${TARGET_MINOR_VERSION}" ]] && [[ "${TARGET_MINOR_VERSION}" -ge "16" ]] ; then
+            local out; out="$(env OC_ENABLE_CMD_UPGRADE_STATUS=true oc adm upgrade status)"
```

Comment:
From the test purpose of the new `status` option, is there any test point that we are going to cover in this function? If this function is designed to cover some test point for `status` option, I think it is better to print out some log message like:
```
OCP-XXXX: PASS_or_FAIL
```
So that later, when we need to fill test result for our regression testing, we can easily get the test result from the log.

Reply:
at this stage no, it does not cover specific test point. currently, the devs are looking for feedback for the usefulness of the data displayed by the status command. here in this function, I'm using the command to log any unusual cluster statuses, which are mostly expected to happen when upgrade is failing for some reason, with the goal to help the upgrade ci watchers, and at the same time collect feedback and evaluate if the information displayed by the command was helpful in finding the root cause of upgrade failure.

Reply:
Got it.